api/page-index/page-index.js

```diff
@@ -636,6 +636,42 @@ module.exports = [
                 title: "into_values (for state_agg)",
                 href: "into_values-state_agg",
               },
+              {
+                title: "heartbeat_agg",
+                href: "heartbeat_agg",
```

Comment:
```suggestion
                href: "heartbeat_agg",
                type: "placeholder",
```
Need this for the new-style pages